layouts/index.html

```diff
@@ -196,16 +196,16 @@ <h3 class="mt-6 text-white">Install with Conda</h3>
                     <br>
                     <p><strong>2.</strong> Then quick install RAPIDS with:</p>
                     <pre
-                        class="highlight use-white-copy"><code>conda create --solver=libmamba -n rapids-24.02 -c rapidsai -c conda-forge -c nvidia rapids=24.02 python=3.10 cuda-version=12.0</code></pre>
+                        class="highlight use-white-copy"><code>conda create --solver=libmamba -n rapids-24.04 -c rapidsai -c conda-forge -c nvidia rapids=24.04 python=3.10 cuda-version=12.0</code></pre>
```

Comment:
Let’s use the latest supported versions. Please apply throughout this repo and docs.
```suggestion
                        class="highlight use-white-copy"><code>conda create --solver=libmamba -n rapids-24.04 -c rapidsai -c conda-forge -c nvidia rapids=24.04 python=3.11 cuda-version=12.2</code></pre>
```